### ISSUE TITLE
FastJsonapi.MultiToJson.to_json_method throws an error if `logger` hasn't been required

### ISSUE THREAD

**Issue by xrx** [Netflix/fast_jsonapi#197]
While testing the gem i ran into an issue by calling `#serialized_json`

```
fast_jsonapi-1.1.1/lib/fast_jsonapi/multi_to_json.rb:48:in `logger': uninitialized constant FastJsonapi::MultiToJson::Logger (NameError)
  from ../fast_jsonapi-1.1.1/lib/fast_jsonapi/multi_to_json.rb:89:in `define_to_json'
  from ../fast_jsonapi-1.1.1/lib/fast_jsonapi/multi_to_json.rb:82:in `rescue in to_json'
  from ../fast_jsonapi-1.1.1/lib/fast_jsonapi/multi_to_json.rb:80:in `to_json'
  from ../fast_jsonapi-1.1.1/lib/fast_jsonapi/serialization_core.rb:98:in `to_json'
  from ../fast_jsonapi-1.1.1/lib/fast_jsonapi/object_serializer.rb:62:in `serialized_json'
```

Already submitted a small fix on #196.

**Comment by shishirmk** [comment]
Merged into dev branch

**Comment by shishirmk** [comment]
This should be released in version 1.2 of the gem
